keel-sql/src/main/resources/db/changelog/20200515-add-unique-id-to-event.yml

```diff
@@ -0,0 +1,34 @@
+databaseChangeLog:
+- changeSet:
+    id: add-unique-id-to-event
+    author: fletch
+    changes:
+    - addColumn:
+        tableName: event
+        columns:
+        - name: uid
+          type: varchar(255)
+          constraints:
+            nullable: true
+    - modifySql:
+        dbms: mysql
+        append:
+          value: " first"
```

Comment:
Nice!

Reply:
Yeah, it took me a while to work it out as the built in options in Liquibase didn't work for me and I had to figure out how to do a MySQL native thing.